tutor4points/core/templates/points.html

```diff
@@ -54,23 +56,24 @@ <h2>Pay Tutor</h2>
                 <div class="purchase-amount">
                     <form method='POST' enctype='multipart/form-data'>
                         {% csrf_token %}
-                        <label>Select Tutor</label>
                         <div class="select-tutor">
```

Comment:
It seems like this css class no longer exists and isn't necessary anymore since the original HTML is already replaced with a form, so I think we could remove this div.